src/qt/stattext.cpp

```diff
@@ -61,6 +61,7 @@ bool wxStaticText::Create(wxWindow *parent,
 void wxStaticText::SetLabel(const wxString& label)
 {
     m_qtLabel->setText( wxQtConvertString( label ) );
+    m_qtLabel->adjustSize();
```

Comment:
You almost certainly should call `AutoResizeIfNecessary()` from here instead. If this doesn't work (which would be bad...), you need to at least check for `wxST_NO_AUTORESIZE` and not change the size if it's set (and we really ought to have a unit test for this not quite obvious behaviour...).

Reply:
That makes good sense. I have made the change, and it works fine. Thanks!